core/block_animations.ts

```diff
@@ -122,27 +122,39 @@ export function connectionUiEffect(block: BlockSvg) {
     },
     workspace.getParentSvg()
   );
-  // Start the animation.
-  connectionUiStep(ripple, new Date(), scale);
-}
+  const animation = dom.createSvgElement(
+    Svg.ANIMATE,
+    {
+      'id': 'animationCircle',
+      'begin': 'indefinite',
+      'attributeName': 'r',
+      'dur': '150ms',
+      'from': 0,
+      'to': 25 * scale,
+    },
+    ripple
+  );
+  dom.createSvgElement(
+    Svg.ANIMATE,
+    {
+      'attributeName': 'opacity',
+      'begin': 'animationCircle.begin',
```

Comment:
I don't think this is actually triggering the opacity animation :/ Could you kick it off manually with `beginElement` like how you're triggering the other one?